alacritty/src/event.rs

```diff
@@ -972,13 +983,41 @@ impl Mouse {
     }
 }
 
+/// State of touch.
+#[derive(Debug)]
+pub struct TouchState {
+    pub start_x: f64,
+    pub start_y: f64,
+    pub x: f64,
+    pub y: f64,
+}
+
+impl Default for TouchState {
+    fn default() -> TouchState {
+        TouchState {
+            start_x: Default::default(),
+            start_y: Default::default(),
+            x: Default::default(),
+            y: Default::default(),
+        }
+    }
+}
+
+impl TouchState {
+    /// Returns square of the distance travelled since the touch began.
+    pub fn dist2(&self) -> f64 {
+        (self.start_x - self.x).powi(2) + (self.start_y - self.y).powi(2)
+    }
```

Comment:
Square distance should be irrelevant for this usecase. What matters is if the same cell is pressed or not.

Reply:
I spent a while playing with different settings for the distance. The difficulty is: on my screen, it's very easy for a sloppy tap to start in one cell and jump to another. Some of this is just that I'm a klutz using a small font, but then, many users are klutzes! Having to fine-tune taps because they're not registering is extraordinarily irritating --- the whole point of touch is that it's faster and requires much less thought.

Reply:
The same could be argued about the mouse though and I'm not aware of anyone having trouble with it here. So I'm not sure why the touchscreen would be any different?

Reply:
At the end of the day, I can only report that it's empirically different for me. I realize that's not helpful.